File: C++/SimpleInterest.cpp
#include<iostream>
using namespace std;

int main(){
    float p,r,t,s;
    cout<<"Enter Principal Amount : ";
    cin>>p;
    cout<<"Rate of Interest in % : ";
    cin>>r;
    cout<<"Time (in years) : ";
    cin>>t;
    s=p*r*t/100;
    cout<<"Simple Interest = "<<s;
    return 0;
}

File: Java/SimpleInterest.java
package com.lev;

import java.util.Scanner;

public class SimpleInterest {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter Principal Amount : ");
        float p = input.nextFloat();
        System.out.print("Rate of Interest in % : ");
        float r = input.nextFloat();
        System.out.print("Time (in years) : ");
        float t = input.nextFloat();
        float s = (p*r*t)/100 ;
        System.out.println("Simple Interest : "+s);

    }
}
